pykern/pkunit.py

```diff
@@ -365,29 +366,29 @@ def pkexcept(exc_or_re, *fmt_and_args, **kwargs):
         if isinstance(exc_or_re, type) and issubclass(exc_or_re, BaseException):
             if isinstance(e, exc_or_re):
                 return
-            if not fmt_and_args:
-                fmt_and_args = (
+            _fail_with_args(
```

Comment:
Keep the arguments the same. You can rename `fmt_and_args` to `args`. It could be argued that's a better name for the arg to pkeq and pkne. 

You don't need a function. Just save in a new variable, e.g. `msg`. Then pass that through at the end as before.